extension/src/popup/ducks/accountServices.ts

```diff
@@ -65,9 +65,11 @@ export const createAccount = createAsyncThunk<
 
 export const fundAccount = createAsyncThunk(
   "auth/fundAccount",
-  async (publicKey: string) => {
+  async ({ publicKey }: { publicKey: string }, { getState }) => {
+    const activePublicKey = publicKeySelector(getState() as AppState);
```

Comment:
instead of making every component pass the `activePublicKey` through, I thought it was a bit neater just to fetch this value at the `ducks` level since the redux thunk has access to the store

Reply:
good call